consensus/fork_choice/src/fork_choice.rs

```diff
@@ -466,10 +466,10 @@ where
                 // for lower slots to account for skip slots.
                 .find(|(_, slot)| *slot <= ancestor_slot)
                 .map(|(root, _)| root)),
-            Ordering::Less => Ok(Some(block_root)),
-            Ordering::Equal =>
             // Root is older than queried slot, thus a skip slot. Return most recent root prior
             // to slot.
+            Ordering::Less => Ok(Some(block_root)),
+            Ordering::Equal =>
```

Comment:
There might be something weird about your version of cargo fmt, because this is failing on stable Rust:

https://github.com/sigp/lighthouse/actions/runs/5828995301/job/15807794125?pr=4565

Are you running `cargo fmt --all`?

Reply:
Weird, both `cargo fmt --all` and `cargo fmt` output nothing to me.

Reply:
No, they don't output anything (unless you use `--check`), they just mutate the files

Reply:
Oops, fixed now.